include/aws/crt/Api.h

```diff
@@ -125,12 +125,46 @@ namespace Aws
             /// @private
             static const Io::IsTlsAlpnSupportedCallback &GetBYOCryptoIsTlsAlpnSupportedCallback();
 
+            /**
+             * Gets the static default EventLoopGroup, creating it if necessary.
+             *
+             * @return EventLoopGroup* A pointer to the static default EventLoopGroup
+             */
+            static Io::ClientBootstrap *GetOrCreateStaticDefaultClientBootstrap();
+
+            /**
```

Comment:
Debatable: personally, I'd also like to have `void SetDefaultEventLoopGroup(Io::EventLoopGroup *eventLoopGroup)` so the user could create one with the exact number of threads they desire, set it at program startup, and not worry about passing it around explicitly.

This is debatable. We can add these GetOrCreateDefault functions now and SetDefault functions later. But like, we're doing this now so it's kind of the time...